ckanext/dalrrd_emc_dcpr/templates/dcpr/snippets/request_form.html

```diff
@@ -104,7 +105,24 @@
             information for aiding the NSIF and CSI in their review and moderation process.')) }}
         {% endcall %}
         {% if data.datasets %}
+            <legend>Dataset(s)</legend>
             {% for ds in data.datasets %}
+                {% set dataset_id = 'dataset'+ loop.index|string %}
+                <div class="panel panel-default" style="margin-bottom: 10px; display: block" id="{{ dataset_id }}">
+                 <div class="panel-heading">
+                    <h4 class="panel-title">
+                                {{ ds.proposed_dataset_title }}
+                            <button
+                                    type="button"
+                                    class="btn btn-danger btn-sm pull-right"
+                                    data-module="remoceDcprRequestDatasets"
```

Comment:
remember to update the `data-module` value after you fix the typo in the name of the module (as mentioned in a comment above)